resources/lang/es/entities.php

```diff
@@ -214,4 +214,26 @@
     'profile_not_created_pages' => ':userName no ha creado ninguna página',
     'profile_not_created_chapters' => ':userName no ha creado ningún capítulo',
     'profile_not_created_books' => ':userName no ha creado ningún libro',
+
+    /**
+     * Comments
+     */
+    'comment' => 'Comentario',
```

Comment:
Used Google translate to translate this. Don't think all these languages are properly translated.